StephaneGrabsia/uQuizz
# Fixing issue with solutions

This PR fix an issue that we noticed on the code submissions: when several exercises were present, the is_correct variable wasn't correct...

This was due to two main issues: 

- The request wasn't sending the right exercise number
- The back wasn't looking to the right solution when checking if it was correct

To test this PR you should : 

1. Launch the docker and go to localhost:8080
2. Create a teacher, a classroom, two exercises and a student
3. Connect yourself to the student main page, and try to answer the two exercises, and check if the results are corrects
